src/scss/_mixins.scss

```diff
@@ -3,6 +3,10 @@
 
 	text-align: center;
 
+	.o-normalise-visually-hidden {
```

Comment:
In Origami components CSS classes are usually namespaced to the component, to avoid conflicts, can we rename this `.o-subs-card-visually-hidden`? 

Aside: We actually deleted the Origami Component Specification - including the CSS namespace requirement - so in the future we have more flexibility e.g to output utility styles once, but we'll probably want to tackle that as a seperate project rather than repeating utilities with the same class name :)

Reply:
This makes sense I'll do this change.